YubiKit/YubiKit/Connections/Shared/Sessions/PIV/YKFPIVSession.m

```diff
@@ -433,7 +433,11 @@ - (void)getCertificateInSlot:(YKFPIVSlot)slot completion:(nonnull YKFPIVSessionR
             NSData *certificateData = [[YKFTLVRecord sequenceOfRecordsFromData:objectData] ykfTLVRecordWithTag:YKFPIVTagCertificate].value;
             CFDataRef cfCertDataRef =  (__bridge CFDataRef)certificateData;
             SecCertificateRef certificate = SecCertificateCreateWithData(nil, cfCertDataRef);
-            completion(certificate, nil);
+            if (certificate != nil) {
+                completion(certificate, nil);
+            } else {
+                completion(nil, [[NSError alloc] initWithDomain:YKFPIVErrorDomain code:YKFPIVFErrorCodeDataParseError userInfo:@{NSLocalizedDescriptionKey: @"Malformed certificate."}]);
+            }
```

Comment:
attestKeyInSlot: has a very similar section of code, but uses descriptionKey "Failed to parse certificate." Should this use the same description?